server/service/mdm.go

```diff
@@ -405,6 +405,28 @@ func (svc *Service) VerifyMDMWindowsConfigured(ctx context.Context) error {
 	return nil
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// Apple or Windows MDM Middleware
+////////////////////////////////////////////////////////////////////////////////
+
+func (svc *Service) VerifyMDMAppleOrWindowsConfigured(ctx context.Context) error {
```

Comment:
The reason I did not just call the existing `VerifyMDMAppleConfigured` and `VerifyMDMWindowsConfigured` and failed only if both failed is because as soon as one fails, it calls `SkipAuthorization` which we don't want if at least one is configured.